docs/develop/documentation.rst

```diff
@@ -0,0 +1,374 @@
+    :func:`fread(input) <datatable.fread>`
+
+If repeating the ``datatable.`` part is tedious, then you can add the following
+declaration at the top of the page::
+
+    .. py:currentmodule:: datatable
+
+
+Note that some of these links may render in red. It means the documentation for
+the referenced function/class/object is missing and still needs to be added:
+:func:`datatable.missing_function()`.
+
+
+Advanced directives
+-------------------
+
+All rst documents are arranged into a tree. All non-leaf nodes of this tree
+must include a ``.. toctree::`` directive, which may also be declared hidden::
+
+    .. toctree::
+        :hidden:
+
+        child_doc_1
+        Explicit name <child_doc_2>
+
+
+The ``.. image::`` directive can be used to insert an image, which may also be
+a link::
+
+    .. image:: <image URL>
+        :target: <target URL if the image is a link>
+
+
+In order to note that some functionality was added or changed in a specific
+version, use::
+
+    .. versionadded:: 0.10.0
+
+    .. versionchanged:: 0.11.0
+
+
+The ``.. seealso::`` directive adds a Wikipedia-style "see also:" entry at the
+beginning of a section. The argument of this directive should contain a link
+to the content that you want the user to see. This directive is best to include
+immeditately after a heading::
+
+    .. seealso:: :ref:`columnsets`
+
+
+
+Changelog support
+-----------------
+
+RST is language that supports extensions. One of the custom extensions that we
+use supports maintaining a changelog. First, the ``.. changelog::`` directive
+which is used in ``releases/vN.N.N.rst`` files declares that each of those
+files describes a particular release of datatable. The format is as follows::
+
+    .. changelog::
+        :version: <version number>
+        :released: <release date>
+        :wheels: URL1
+                 URL2
+                 ...
+
+        changelog content...
+
+        .. contributors::
+
+            N @username <full name>
+            ...
+            --
+            N @username <full name>
+            ...
+
+
+The effect of this declaration is the following:
+
+- The title of the page is automatically inserted, together with an anchor
+  that can be used to refer to this page;
+
+- A Wikipedia-style infobox is added on the right side of the page. This
+  infobox contains the release date, links to the previous/next release,
+  and the links to all wheels that where released at that version. The wheels
+  are grouped by the python version / operating system. An sdist link may also
+  be included as one of the "wheels".
+
+- Within the ``.. changelog::`` directive, a special form of list items is
+  supported::
+
+      -[new] New feature that was added
+
+      -[enh] Improvement of an existing feature or function
+
+      -[fix] Bug fix
+
+      -[api] API change
+
+  In addition, if any such item ends with the text of the form ``[#333]``,
+  then this will be automatically converted into a link to a github issue/PR
```

Comment:
May be surround `/` with spaces just like everywhere else.

Reply:
According to [Chicago Manual of Style](https://www.chicagomanualofstyle.org/qanda/data/faq/topics/Punctuation/faq0096.html) there should be no spaces when `/` separates individual words (as opposed to sentences).

Reply:
I see, then let's make it be consistent with this manual in the other places like this one https://github.com/h2oai/datatable/pull/2492/files#diff-dfe271025faea4bfbd4bddd37499bcd7R285 ?